my submission to the challenge

procesorplus opened the pull request:
Hi, I've added my page to the submissions folder. Thanks.

MattCSmith commented:
Looks good @procesorplus, I'm going to merge this, but be sure to create a PR adding your entry to the Readme file as per the instructions 🙂

procesorplus commented:
Hi Matt - how do I do it? :) I have added the entry by editing the readme file on my fork... I have also added the entry with Sublime to the readme on my branch and sent it over, but it showed the conflict... I am lost :/ I'd appreciate a hint
